packages/edit-post/src/components/sidebar/plugin-sidebar/index.js

```diff
@@ -136,6 +136,7 @@ export default compose(
 		return {
 			icon: ownProps.icon || context.icon,
 			sidebarName: `${ context.name }/${ ownProps.name }`,
+			priority: ownProps.priority || context.priority,
```

Comment:
It doesn't seem to be used in the implementation of the sidebar's slot.

Reply:
Removed. I think this was an earlier attempt - thanks for catching it!